examples/wordle/package.json

```
@@ -16,6 +16,7 @@
     "vite": "^2.9.13"
   },
   "dependencies": {
+    "mirotone": "^4.3.0",
```

```suggestion
    "mirotone": "4",
```

When we dont need to bother with updating for minor updates